api/go.sum

```diff
@@ -437,6 +437,7 @@ golang.org/x/sys v0.0.0-20210315160823-c6e025ad8005/go.mod h1:h1NjWce9XRLGQEsW7w
 golang.org/x/sys v0.0.0-20210320140829-1e4c9ba3b0c4/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
 golang.org/x/sys v0.0.0-20210330210617-4fbd30eecc44/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
 golang.org/x/sys v0.0.0-20210403161142-5e06dd20ab57/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
+golang.org/x/sys v0.0.0-20210510120138-977fb7262007 h1:gG67DSER+11cZvqIMb8S8bt0vZtiN6xWYARwirrOSfE=
```

Comment:
Why is that? Followed [this recommendation](https://github.com/golang/go/wiki/Modules#releasing-modules-all-versions) because I didn't know whether to commit `go.sum` either.

Reply:
When you have your dev environment, you'll notice that in a lot of the `go.mod` files, we have some things like `replace kyaml => ./kyaml`, that's because we usually have all our modules pointing to each other. When we release, we remove these replace statements so that the modules are pointed at pinned versions.

This adding/removing of replace statements creates a lot of noise in the go.sum files and we usually don't try to deal with it until we release. 

(Side note, this setup may change if we start to use Go 1.18 workspaces)

Reply:
If I'm understanding you correctly, my new entry to the `go.sum` files would add more unnecessary clutter. The better solution is just to add these entries before a release?

Reply:
It's not that your new go.sum entry is unnecessary clutter necessarily, but more that the go.sum file itself in the PR is large and not really useful to review. Since we pin and unpin so often and therefore need to modify go.sum files during release, it's not important to add the go.sum file here.

Reply:
Ahh, makes sense. Thank you for the explanations!

Reply:
Yeah, if it causes errors then we should add them back in.